src/services/event-processor.service.js

```diff
@@ -1,7 +1,24 @@
 const { quotesConstants } = require('../constants/quote-constants')
 const { transferConstants } = require('../constants/transfer-constants')
 const { settlementConstants } = require('../constants/settlement-constants')
-const { eventType } = require('../constants/event-types')
+const { eventTypes } = require('../constants/event-types')
+const { getSettlementReportParams } = require('../custom-transformations/settlement-report-params')
+
+const _getReportingParams = (msg, eventType) => {
+  switch(eventType) {
```

Comment:
I suppose this method should be inside `EventProcessorService` class

Reply:
I tried that, but getting few errors. So it's working this way and left like this.